wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -68,6 +68,13 @@
   public static final String SERVER_EXEC_AGENT_WEBSOCKET_REFERENCE = "exec-agent/ws";
 
   public static final String WS_AGENT_PORT = "4401/tcp";
+  public static final String WS_AGENT_TRACK_CONNECTION_SUBSCRIBE =
+      "ws-agent/track/connection/subscribe";
+  public static final String WS_AGENT_TRACK_CONNECTION_UNSUBSCRIBE =
+      "ws-agent/track/connection/unsubscribe";
+  public static final String WS_AGENT_TRACK_CONNECTION_HEARTBEAT =
+      "ws-agent/track/connection/heartbeat";
+  public static final int WS_AGENT_TRACK_CONNECTION_PERIOD = 5_000; // milliseconds
```

Comment:
Maybe `WS_AGENT_TRACK_CONNECTION_PERIOD_MILLIS` in order to make constant's name clearer and avoid redundant comment?

Reply:
thank you, I've renamed constants and hope constants' names more clearer now